### PR TITLE
Update example .yaml file in README

### PR DESCRIPTION
**Pull request by ghost**
-Change file name from `example/armada.yaml` to `example/openstack-helm.yaml`

**What is the purpose of this pull request?**: Update the name of the example .yaml file in the README

**What issue does this pull request address?**: 

**Notes for reviewers to consider**:

**Specific reviewers for pull request**:

@gardlt

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -67,7 +67,7 @@ Installing Dependecies:
 
 you can run:
 
--  ``tox testenv:ubuntu`` or ``sudo sh scripts/libgit2.sh``
+-  ``tox testenv:ubuntu`` or ``sudo sh tools/libgit2.sh``
```
Comment (gardlt) [inline README.rst]: if you are updating this, can you also update the tox command.

Comment (theyer) [inline README.rst]: done

**Comment by gardlt** [comment]
@theyer LGTM

**Review by gardlt — APPROVED** [review]
LGTM

**Pull request merged**